#!/usr/bin/env python
PACKAGE = "quadruped_locomotion"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("k_joints_p", double_t, 0, "P-gain for joint torque controller", 0, 1.0, 10)
gen.add("k_joints_d", double_t, 0, "P-gain for joint torque controller", 0, 0.1, 10)

exit(gen.generate(PACKAGE, "joint_controller", "DynamicControl"))
